hook/hooktest/service.go

```diff
@@ -20,22 +20,15 @@ import (
 // It should be buffered with a size of at least 2.
 func NewServiceFunc(r *Runner, notify chan ServiceEvent) func(service.OSServiceParams) service.OSService {
```

Comment:
I think this would be more obvious when it's easy to look up the definition of OSService and OSServiceParams, which have appropriate doc comments.